user-manual/transfer/forensic.rst

```diff
@@ -4,130 +4,169 @@
 Forensic disk images
 ====================
 
-Workflow
---------
+Archivematica supports the preservation of forensic disk images. Selecting the
+disk image :ref:`transfer type <transfer-types>` is not required to preserve
+disk images - you can use the standard transfer type or the bag transfer types,
+if your disk image is also bagged. However, the disk image transfer type does
```

Comment:
This is nit-picky but I think we can drop the "However" and "does" here -- I feel like it makes the paragraph less 'on-the-defense' and more generally assertive.